doc/man3/SSL_set_connect_state.pod

```diff
@@ -2,7 +2,8 @@
 
 =head1 NAME
 
-SSL_set_connect_state, SSL_set_accept_state - prepare SSL object to work in client or server mode
+SSL_set_connect_state, SSL_set_accept_state,
+SSL_is_server - prepare SSL object to work in client or server mode
```

Comment:
I'd suggest leaving the original line alone and adding a new entry for `SSL_is_server`.